### DISCUSSION TITLE
A way to share/open-in another app?

### DISCUSSION THREAD

**Discussion by 3d73a4a412** [smplayer-dev/smplayer#232] [Q&A]
SMPlayer is great for playing videos, and the playlist really helps, what I was wondering is if there is a way to share/open-in/send-to a video with another app? This use case is something like vidcutter when i am going through videos and want to cut the cruft out of some videos to be able to easily just right click (or hotkey or whatever) and open that video, from smplayer, in vidcutter. Is this possible?
Thanks!

**Comment by Cootmaster** [comment]
yes you can set  SM TUBE open with  UNDER PLAYERS 
![image10](https://user-images.githubusercontent.com/80638263/136406548-20689f7a-0ba6-41ac-adae-749e94f7e9c1.jpg)
like this then play with  with  to  launch vid cutter
i tried lossless  cut  it launches  but nothing to play  .  
hopefully that was your question
here is a  example with vlc
i believe its  LAUNCH TO PLAY  ONLY

**Reply by 3d73a4a412** [reply]
Thanks for the reply!
So to make sure I understand (sorry, smp noob), can I play a playlist of local videos with smtube?
I installed smtube and made vid cutter a player, but it *seemed* that I was only able to open/browse youtube videos with smtube, not local videos?

**Comment by Cootmaster** [comment]
in SMTUBE  
you can click on a video   link COPY  then PLAY WITH  ..
this will set up  multiple players like VLC  
then PLAY WITH  will  launch  VLC  then open vlc and automatically start playing
ive used  several  downloader  programs that launch  with PLAY WITH  after  copying the video link  
YES  you cant  OPEN  from SM TUBE
my suggestion  
is  download a video with  sm tube then open with some software  like any video  converter  then set the parameters to auto launch and auto down load like in picture  above  or
 just add a video into the cutter from your C  drive  
i use video cutter  and  lossless cut  BOTH ACCESSED  through   C drive or where ever you saved it 

SMPLAYER  is used to PLAY videos  from a  local drive
you CAN make 
play lists IN SMPLAYER 

again id have your videos on c or what ever then load it into your cutter  then edit
idont think you can edit INSIDE  smtube or smplayer
